### DISCUSSION TITLE
Highlight the missing field to be filled while submitting a form

### DISCUSSION THREAD

**Discussion by umutyorulmaz** [lowcoder-org/lowcoder#1402] [Q&A] [answered]
I am creating a form. When the submit button is pressed, is there a way to highlight the empty field that required to be filled ? For example red colour can show up around or in the missing field signalling it to the user ?

**Comment by FalkWolsky** [comment, marked as answer]
This can be done by the help of event handlers and a small trick in the color input field for example the label or border of the field.
In all color and many other property fields you will recognise a small "js" at the very right side. This opens an input field and here you can work with dynamic binding of {{...}} so you can dynamically adapt the value up to your interaction needs. In our youtube channel we will publish on 23.01.2025 a new tutorial video - showing exactly this. https://youtu.be/OagWRQmPjiA (video will become active then) The channel: https://www.youtube.com/watch?v=vy68s2B92X0&list=PLhKId1R4AABH92VsU2OYQmRAoYuMlmmM4

**Reply by umutyorulmaz** [reply]
thank you so much looking forward for the video
